src/app/libp2p_helper/src/codanet.go

```diff
@@ -91,23 +97,26 @@ func isPrivateAddr(addr ma.Multiaddr) bool {
 }
 
 type CodaConnectionManager struct {
-	ctx              context.Context
-	host             host.Host
-	p2pManager       *p2pconnmgr.BasicConnMgr
-	minaPeerExchange bool
-	getRandomPeers   getRandomPeersFunc
-	OnConnect        func(network.Network, network.Conn)
-	OnDisconnect     func(network.Network, network.Conn)
-}
-
-func newCodaConnectionManager(maxConnections int, minaPeerExchange bool) *CodaConnectionManager {
+	ctx                context.Context
+	host               host.Host
+	p2pManager         *p2pconnmgr.BasicConnMgr
+	minaPeerExchange   bool
+	getRandomPeers     getRandomPeersFunc
+	OnConnect          func(network.Network, network.Conn)
+	OnDisconnect       func(network.Network, network.Conn)
+	consensusNodeCount int64
+	maxConsensusNode   int64
```

Comment:
it might be nicer to have this as a minimum instead, since we probably want the # of consensus nodes connected to be above a certain amount

Reply:
Done